packages/styles/src/placeholders/_modes.scss

```diff
@@ -0,0 +1,10 @@
+@use '@swisspost/design-system-tokens/mode' as mode;
+@use './../mixins/tokens';
+
+%post-light {
```

Comment:
What about using the name `%mode-light` instead?

Reply:
Good idea, I think I'll call it `%color-mode-light` which would be even closer to the selector.